modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -100,28 +100,49 @@ spec:
       defaultPlugins:
       - openshift
       - aws
-    restic:
+    nodeAgent: <1>
       enable: false
+      uploaderType: restic
   snapshotLocations:
     - velero:
         config:
-          credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials <1>
-          enableSharedConfig: "true" <2>
-          profile: default <3>
-          region: ${REGION} <4>
+          credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials <2>
+          enableSharedConfig: "true" <3>
+          profile: default <4>
+          region: ${REGION} <5>
         provider: aws
 EOF
 ----
 
+<1> If you are using OADP 1.2, replace this configuration:
 +
```

Comment:
```suggestion
EOF
----
<1> If you are using OADP 1.2, replace this configuration:
+
```
Indentation looks slightly off here for the callouts, maybe because there's a line space here